src/styles/class-overrides.scss

```diff
@@ -123,6 +121,26 @@
   }
 }
 
+.banner__language-selector-label {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.usa-nav__submenu .usa-nav__submenu-item a {
+  text-decoration: none;
+  &:hover {
+    text-decoration: none;
```

Comment:
Do you have to repeat this line?

Reply:
Good call - seems like it's not needed here